src/library_exceptions.js

```diff
@@ -110,6 +110,7 @@ var LibraryExceptions = {
     };
   },
 
+#if !DISABLE_EXCEPTION_CATCHING
```

Comment:
what error happens without this change?

Reply:
Undefined symbol `__cxa_is_pointer_type` below.

Reply:
I'm happy to debug further if you like.. but it seemed natural to me that `CatchInfo` should only be defined when exception catching is enabled.

Reply:
Ah - I think the issue is that the file gets included at all. `modules.js` will not include this file if exceptions are disabled, so these ifdefs should be redundant. Is the test perhaps using INCLUDE_FULL_LIBRARY? Then perhaps we need to make that not include files that do not make sense (like this file without exceptions).

Reply:
From modules.js:

```
     if (!DISABLE_EXCEPTION_THROWING) {                                                                                      
        libraries.push('library_exceptions.js');                                 
      } else {                                                                   
        libraries.push('library_exceptions_stub.js');                            
      }     
```

It looks like only DISABLE_EXCEPTION_THROWING=1 will cause the stubs to be included.. DISABLE_EXCEPTION_CATCHING=1 (which is the default) does not use the stubs.  Instead it includes this file.

Reply:
Should `DISABLE_EXCEPTION_CATCHING=1` have its own stubs file?

Reply:
I spit this part of the change into https://github.com/emscripten-core/emscripten/pull/13862 so we can continue the discussion there.